src/model.cpp

```diff
@@ -1265,9 +1281,17 @@ void Model::fdydp(const realtype t, const AmiVector &x) {
 
     /* get dydp slice (ny) for current time and parameter */
     for (int ip = 0; ip < nplist(); ip++)
-        fdydp(&dydp_.at(ip * ny), t, x.data(), state_.unscaledParameters.data(),
-              state_.fixedParameters.data(), state_.h.data(), plist(ip), w_.data(),
-              dwdp_.data());
+        if (pythonGenerated) {
+            fdydp(&dydp_.at(ip * ny), t, x.data(),
```

Comment:
At some point we may want to change those signatures to take spans... Slightly better readable without all those `.data()`...